### PR TITLE
Accessibility corrections reported by axe part 2

### PR DESCRIPTION
**Pull request by dmundra**
Fixes:
* Switching the people headers from ``<h3>`` to ``<h2>``. The CSS styling shouldn't affect this change.
* Switching ``<main>`` to ``<div>`` for certain pages that will have an outer main.
* Placing home page ``<section>`` in an ``<aside>`` following directions from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

### CODE REVIEW

**Review by mgifford — COMMENTED** [review]
Notices some spacing issues when looking at the files. I think they aren't being consistent. Are we using spaces or tabs here? 

Opportunity to standardize I think.

**Comment by dmundra** [comment]
> Notices some spacing issues when looking at the files. I think they aren't being consistent. Are we using spaces or tabs here?
> 
> Opportunity to standardize I think.

I am using spaces.

**Comment by dmundra** [comment]
@mgifford pa11y-ci axe tests are green! https://github.com/CivicActions/accessibility/pull/137#issuecomment-796975773

File: pages/people.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                         </div>
                     </div>
                     <header class="usa-card__header">
-                        <h3 class="usa-card__heading"><a href="{{ site.baseurl }}{{ author.url }}">{{ author.name }}</a></h3>
+                        <h2 class="usa-card__heading"><a href="{{ site.baseurl }}{{ author.url }}">{{ author.name }}</a></h2>
```
Comment (dmundra) [inline pages/people.html:30]: Ignoring remark-link in this case because <h1> does exist on this page.

**Review by mgifford — APPROVED** [review]
It wasn't a serious issue, just an observation. We can fix it later if it is still needed.

**Pull request merged**